docs/application/web/guides/w3c/supplement/webgl.md

```diff
@@ -803,7 +805,7 @@ You can apply a texture and 3D perspective with touch events by using the [glMat
        attribute vec3 attVertexPos; /* Vertex coordinate attribute */
        attribute vec2 attTextureCoord; /* Texture coordinate attribute */
 
-       uniform mat4 unifMVMatrix; /* Model–view uniform */
+       uniform mat4 unifMVMatrix; /* Model-view uniform */
        uniform mat4 unifPMatrix; /* Projection matrix uniform */
 
        /* Texture coordinate storing variable used in the fragment shader */
```

Comment:
line 924,925: Add 3 spaces at the beginning.

Reply:
The requested `–` creates an error on Firefox, so it is changed.
![image](https://user-images.githubusercontent.com/9958374/41215690-803b4c32-6d8c-11e8-888f-7d77655369e0.png)